tests/testthat/test-nonASCII.R

```diff
@@ -0,0 +1,18 @@
+context("nonASCII")
+
+test_that("can generate nonASCII document", {
+  test_pkg <- temp_copy_pkg('testNonASCII')
+  on.exit(unlink(test_pkg, recursive = TRUE))
+
+  expect_output(roxygenize(test_pkg), "printChineseMsg[.]Rd")
+  expect_true(file.exists(file.path(test_pkg, "man", "printChineseMsg.Rd")))
+
+  cnChar <- readLines(file.path(test_pkg, "man", "printChineseMsg.Rd"), encoding = "UTF-8")
+
+  # Because the parse in testthat::test don't specify encoding to UTF-8 as well,
+  # we have to use `stringi::stri_escape_unicode` to avoid errors.
+  expect_true(
+    any(grepl("\u6211\u7231\u4e2d\u6587", cnChar)) &&
```

Comment:
Why does one string get wrapped with `enc2utf8()` and not the other? A comment would be helpful.